src/components/DependsOn.jsx

```diff
@@ -105,6 +105,7 @@ function DependsOn(props) {
             size={dependence.Size}
             digest={dependence.Digest}
             key={index}
+            vulnerabiltySeverity={dependence.Vulnerabilities.MaxSeverity}
```

Comment:
we should add the '?' operator after vulnerabilities, so the app won't break if some data is missing